lms/djangoapps/courseware/module_render.py

```diff
@@ -642,6 +643,8 @@ def rebind_noauth_module_to_user(module, real_user):
 
     system.set(u'user_is_staff', user_is_staff)
     system.set(u'user_is_admin', has_access(user, u'staff', 'global'))
+    system.set(u'user_is_beta_tester', CourseBetaTesterRole(course_id).has_user(user))
```

Comment:
@cpennington I was asking about this ```CourseBetaTesterRole(course_id).has_user(user)```, I used this function because I'm unable to find any other example in the code to do that.

My question was that to check beta tester access can we use ```has_access()```? but i don't think there is a functionality like this for beta tester, so should we create a functionality like this or use the above as it is?